test/support/test_routes.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Maverick.TestRoute1 do
   use Maverick, scope: "/route1"
 
-  @route path: "multiply", args: {:required_params, [:num1, :num2]}, error: 403
+  @route path: "multiply", args: [:params], error: 403
```

Comment:
this is transitional until you get to the more explicit "required non-nil values" you mentioned?

Reply:
Yes, want to get basic argument handling done, and then come back and add required params.